### DISCUSSION TITLE
Is Bluelinky still maintained?

### DISCUSSION THREAD

**Discussion by Jens5395** [Hacksore/bluelinky#217] [Q&A]
Two weeks ago I raised an issue about my 'car location' function no longer working. There has been no reply to the issue. I am wondering if I just need to be more patient for a reply or if Bluelinky is no longer maintained .....

**Comment by Hacksore** [comment]
I'd say the biggest challenge is that each region has a slightly different API where we need someone from that region to help troubleshoot.

Since you are in Canada we'd need someone who has the ability to help diagnose the issue and see if it's a problem in the bluelinky library code or the node-red addon code.

Also, we are pretty limited on the number of maintainers for each region and I don't even own a Hyundai anymore 😬.

**Comment by PierreLevres** [comment]
On Discord, cdnninja offered to look into it. Also there I tried to shed some light on more difficulties with the canadian implementation, hoping it would ring a bell. At the moment it seems stuck at capturing canadian traffic. So, definitely people are looking into to. Meanwhile, did you have a look into the HA-custom component "kia_uvo" ? Does that work well for you?
